README.rst

```diff
@@ -256,3 +256,65 @@ data that the node is no longer responsible for.
 .. _Jolokia: https://jolokia.org/
 .. _STUPS Cassandra: https://github.com/zalando/stups-cassandra
 .. _Più: http://docs.stups.io/en/latest/components/piu.html
+
+Upgrade your cluster from Cassandra 2.1 -> 3.7
+===================
+
+In order to upgrade your Cluster you should run the following steps. You should have in mind that this process is a rolling update, which means applying the changes for each node in your cluster one by one.
+After upgrading the last node in your cluster you are done.
+
+**Before you actually start, you should read the [Datastax guide](https://docs.datastax.com/en/latest-upgrade/upgrade/cassandra/upgrdCassandraDetails.html) and consider the upgrade restrictions.**
+
+1. Check for the latest Cassandra version: 
+  `curl https://registry.opensource.zalan.do/teams/stups/artifacts/planb-cassandra-3/tags | jq '.[-1].name'`
+2. Connect to the instance where you want to run the upgrade and enter your docker container. 
+3. Run `nodetool upgradesstables` and `nodetool drain`. The latter command will flush the memtables and speed up the upgrade process later on. *This command is mandatory and cannot be skipped.*
+4. Stop the docker container and remove it
+5. If you are running cassandra with the old folder structure where the data is directly located in __/var/lib/cassandra/__ do the following. **If not go on with step 6**. 
+  1.   Move all keyspaces to /var/lib/cassandra/data/data 
```

Comment:
You are using docker path to the data volume here, but below in example you show Taupage's path with `/mounts/...`.  It would make sense to unify it.  Since the docker container was stopped at previous step, it makes sense to stick with the latter one.